content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

```diff
@@ -1,80 +1,173 @@
-was narrowed down to only focus on the Leads (or Opportunities) whose :guilabel:`Tag Name` contain
-a value of `product` and they *also* have an email address set up in the database.
+Click :guilabel:`Edit` to open the :guilabel:`Open: Activities` pop-up form for that specific
+activity, in which that activity can be modified. Click the :guilabel:`Delete` button to completely
+delete that specific activity from the workflow.
 
-Beneath the filter nodes, Odoo will display how many records in the database fall into the
-previously-specified criteria filter. In this example, there are 18 records in the database that
-match the criteria.
+Activity graph tab
+------------------
 
-.. image:: understanding_metrics/marketing-automation-lead-filters.png
+By default, in every activity block, the :guilabel:`graph` tab of the activity block is selected
+(represented by a :guilabel:`pie chart` icon), displaying related metrics as a simple line graph.
+The success metrics are represented in `green` and the rejected metrics are represented in `red`.
+
+A numerical representation of both (:guilabel:`Success` and :guilabel:`Rejected`) is presented to
+the right of the line graph, as well.
+
+.. tip::
+   Hovering over any point in the line graph of the activity block reveals a notated breakdown of
+   data for that specific date.
+
+Beneath the graph in the activity block, *if* the activity type is *Email* or *SMS*, there is a line
+of accessible data figures that are directly related to the campaign activity. These detailed
+statistics provide a bird's eye view of the following metrics: :guilabel:`Sent` (numerical),
+:guilabel:`Clicked` (percentage), :guilabel:`Replied` (percentage), and :guilabel:`Bounced`
+(percentage).
+
+.. tip::
+   Clicking any of those stats on the :guilabel:`DETAILS` line, beneath the line graph, reveals a
+   separate page containing every specific record for that particular data point.
+
+Activity filter tab
+-------------------
+
+Next to the :guilabel:`graph` tab on the activity block, there's the option to open a
+:guilabel:`filter` tab (represented by a :guilabel:`filter/funnel` icon).
+
+.. image:: understanding_metrics/activity-filter-tab.png
    :align: center
-   :alt: Lead generation filters used to refine records on marketing automation campaigns in Odoo.
+   :alt: What a campaign activity filter tab looks like in Odoo Marketing Automation.
 
-At the top of the marketing automation form, there are a series of smart buttons. The
-:guilabel:`Participations` smart button in this example indicates that, out of those 18 records, 25
-have become participants. In other words, they have matched the criteria, and have already
-interacted with the campaign in one way or another.
+Clicking the :guilabel:`filter` tab on an activity block, reveals what the specific filters are for
+that particular campaign activity, and how many records in the database match that specific
+criteria.
```

Comment:
What are some examples of filters people would want to apply to get the marketing campaign info they need?

This information isn't necessary for you to add here, but was just something I was personally curious about (: